.ci/oci-nightly-olm.sh

```diff
@@ -17,46 +17,71 @@
 set -e
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
-# error on unset variables
-set -u
 
 export OPERATOR_REPO=$(dirname $(dirname $(readlink -f "$0")));
 source "${OPERATOR_REPO}"/.github/bin/common.sh
 source "${OPERATOR_REPO}"/.github/bin/oauth-provision.sh
 
-#Stop execution on any error
+# Stop execution on any error
 trap "catchFinish" EXIT SIGINT
 
-overrideDefaults() {
-  # CI_CHE_OPERATOR_IMAGE it is che operator image builded in openshift CI job workflow. More info about how works image dependencies in ci:https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
-  export OPERATOR_IMAGE=${CI_CHE_OPERATOR_IMAGE:-"quay.io/eclipse/che-operator:nightly"}
+deployChe() {
+  chectl server:deploy  --telemetry=off --workspace-engine=dev-workspace --platform=openshift --installer=operator --batch
 }
 
-runTests() {
-    # Deploy Eclipse Che applying CR
-    applyOlmCR
-    waitEclipseCheDeployed "nightly"
-    provisionOAuth
-    startNewWorkspace
-    waitWorkspaceStart
-
-    # Dev Workspace controller tests
-    deployDevWorkspaceController
-    waitDevWorkspaceControllerStarted
-
-    sleep 10s
-    createWorkspaceDevWorkspaceController
-    waitWorkspaceStartedDevWorkspaceController
-
-    sleep 10s
-    createWorkspaceDevWorkspaceCheOperator
-    waitWorkspaceStartedDevWorkspaceController
+runTest() {
+  deployChe
+  waitDevWorkspaceControllerStarted
+
+  # wait for 2 min for devworkspace-controller ready to work.
+  sleep 120
```

Comment:
Why sleep in this case?

Reply:
I thoughtt waitDevWorkspaceControllerStarted wait for devworkspace to be deployed

Reply:
waitDevWorkspaceControllerStarted works incorrectly. It finishes in waiting for "controller.devfile.io" webhook but devworkspace pods still not in Running status. And `oc apply -f https://raw.githubusercontent.com/devfile/devworkspace-operator/main/samples/flattened_theia-next.yaml -n eclipse-che` command fails(you can check it [here).](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/eclipse-che_che-operator/727/pull-ci-eclipse-che-che-operator-master-v7-olm-nightly-deployment/1373903576975282176/artifacts/olm-nightly-deployment/olm-nightly-deployment/build-log.txt)